sematic/resolvers/tests/test_cloud_resolver.py

```diff
@@ -27,42 +30,34 @@ def pipeline() -> float:
 
 
 @mock.patch("socketio.Client.connect")
-@mock.patch("sematic.resolvers.cloud_resolver.get_image_uri")
+@mock.patch("sematic.resolvers.cloud_resolver.get_image_uri", new=lambda: "some_image")
```

Comment:
TIL about `new` in `@mock.patch` 👍

Reply:
Since we are not processing any input, `return_value` would do?

Reply:
Done.

Reply:
This was causing the tests to fail. Apparently the `return_value` parameter is ignored by `mock.patch`.
[The API](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) doesn't explicitly list it, and apparently it isn't being handled in `**kwargs`.
Reverted.